### ISSUE TITLE
Instruments should take intregration weights

### ISSUE THREAD

**Issue by Jacfomg** [qiboteam/qibolab#707]
That should improve the classification and signal acquisition

**Comment by stavros11** [comment]
This already works for Zurich and the general platform (loading/dumping), we just need to propagate to qblox, QM, etc. right?

**Comment by Jacfomg** [comment]
Yup, if they support them I feel like they would be an improvement. I think they are not the biggest priority right now though.
